src/index.html

```diff
@@ -1,13 +1,305 @@
+            <p class="details__description">
+              The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.
+            </p>
+          </article>
+
+          <article class="details__article">
+            <div class="details__img-container">
+              <div class="details__img-wrapper">
+                <img
+                  src="img/image3.png"
+                  alt="#"
+                  class="details__img"
+                >
+              </div>
+
+              <div class="details__img-wrapper">
+                <img
+                  src="img/image4.png"
+                  alt="#"
+                  class="details__img"
+                >
+              </div>
+            </div>
+            <h3 class="details__article-header">
+              Range & Integrated lights
+            </h3>
+            <p class="details__description">
+              The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </p>
+          </article>
+
+          <article class="details__article">
+            <div class="details__img-container">
+              <div class="details__img-wrapper">
+                <img
+                  src="img/image5.png"
+                  alt="#"
+                  class="details__img"
+                >
+              </div>
+
+              <div class="details__img-wrapper">
+                <img
+                  src="img/image6.png"
+                  alt="#"
+                  class="details__img"
+                >
+              </div>
+            </div>
+            <h3 class="details__article-header">
+              Hydraulic disc brakes & Lightweight
+            </h3>
+            <p class="details__description">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </p>
+          </article>
+        </div>
+        <div class="details__link">
+          <a href="#" class="button">Explore</a>
+        </div>
+      </section>
+
+      <section class="contacts" id="contacts">
+        <h2 class="contacts__header">Contact us</h2>
+        <div class="contacts__wrapper">
+          <section class="contacts__input-section">
+            <form action="#" class="contacts__form">
+              <input
+              type="text"
+              class="contacts__name"
+              placeholder="Name"
+              >
+
+              <input
+              type="text"
+              class="contacts__email-input"
+              placeholder="Email"
+              >
+
+              <textarea
+              placeholder="Message"
+              name="coments"
+              class="contacts__message"
+              ></textarea>
```

Comment:
It would be great if you can add some hover effect for your form inputs and textarea
or at least add `cursor: pointer; ` on hover

Reply:
I mean that the mouse should also change...like it work on button right now